packages/editor/src/components/provider/index.native.js

```diff
@@ -303,6 +316,17 @@ class NativeEditorProvider extends Component {
 		return false;
 	}
 
+	onVoiceToContent( { content } ) {
+		const { insertBlocks } = this.props;
+		const blocks = pasteHandler( {
+			plainText: content,
+		} );
+
+		if ( blocks ) {
+			insertBlocks( blocks, undefined, undefined, false );
```

Comment:
From the [implementation](https://github.com/WordPress/gutenberg/blob/af03001802ff0b19de67b22cf8c0556f132c83e4/packages/block-editor/src/store/actions.js#L545) we avoid setting the second and third param (index, rootClientId) so we can set last param `updateSelection` as `false` to not focus in any block automatically.